src/pudl/transform/ferc1.py

```diff
@@ -7,1400 +7,2995 @@
-        "none.",
-        "fuel facilities",
-        "0",
-        "not in service",
-        "none",
-        "common expenses",
-        "expenses common to",
-        "retired in 1981",
-        "retired in 1978",
-        "na",
-        "unit total (note3)",
-        "unit total (note2)",
-        "resp. share (note2)",
-        "resp. share (note8)",
-        "resp. share (note 9)",
-        "resp. share (note11)",
-        "resp. share (note4)",
-        "resp. share (note6)",
-        "conventional",
-        "expenses commom to",
-        "not in service in",
-        "unit total (note 3)",
-        "unit total (note 2)",
-        "resp. share (note 8)",
-        "resp. share (note 3)",
-        "resp. share note 11",
-        "resp. share (note 4)",
-        "resp. share (note 6)",
-        "(see note 5)",
-        "resp. share (note 2)",
-        "package",
-        "(left blank)",
-        "common",
-        "0.0000",
-        "other generation",
-        "resp share (note 11)",
-        "retired",
-        "storage/pipelines",
-        "sold april 16, 1999",
-        "sold may 07, 1999",
-        "plants sold in 1999",
-        "gas",
-        "not applicable.",
-        "resp. share - note 2",
-        "resp. share - note 8",
-        "resp. share - note 9",
-        "resp share - note 11",
-        "resp. share - note 4",
-        "resp. share - note 6",
-        "plant retired- 2013",
-        "retired - 2013",
-        "resp share - note 5",
-        "resp. share - note 7",
-        "non-applicable",
-        "other generation plt",
-        "combined heat/power",
-        "oil",
-    ],
-}
-"""
-A mapping from canonical plant kinds (keys) to the associated freeform strings (values)
-identified as being associated with that kind of plant in the FERC Form 1 raw data.
-There are many strings that weren't categorized, Solar and Solar Project were not
-classified as these do not indicate if they are solar thermal or photovoltaic. Variants
-on Steam (e.g. "steam 72" and "steam and gas") were classified based on additional
-research of the plants on the Internet.
-"""
+        """
+        df = df.copy()
+
+        FuelFix = namedtuple("FuelFix", "fuel from_unit to_unit mult")
+        fuel_fixes = [
+            FuelFix("coal", "mmbtu", "ton", (1.0 / 19.85)),
+            FuelFix("coal", "klbs", "ton", (1.0 / 2.0)),
+            FuelFix("coal", "lbs", "ton", (1.0 / 2000.0)),
+            FuelFix("coal", "btu", "ton", (1.0 / 19.85e6)),
+            FuelFix("oil", "gal", "bbl", (1.0 / 42.0)),
+            FuelFix("oil", "kgal", "bbl", (1000.0 / 42.0)),
+            FuelFix("oil", "ton", "bbl", 7.46),
+            FuelFix("gas", "mmbtu", "mcf", (1.0 / 1.037)),
+            FuelFix("nuclear", "mwdth", "mwhth", 24.0),
+            FuelFix("nuclear", "mmmbtu", "mwhth", (1.0 / 3.412142)),
+            FuelFix("nuclear", "btu", "mwhth", (1.0 / 3412142)),
+            FuelFix("nuclear", "grams", "kg", (1.0 / 1000)),
+        ]
```

Comment:
I liked the idea of these `fuel_fixes` being a part of the table-level transform parameters instead of being hidden within this method. Mostly for transparency/an ability to expose to users all (or many) of the types of transforms we apply to each table. I'm not sure this is a good enough reason to move them, but that's definitely my thoughts.